js/main.js

```diff
@@ -10,18 +10,18 @@ document.querySelector('.openMenu').addEventListener('click', displayMobileMenu)
 closeMobileItems.forEach(((element) => element.addEventListener('click', closeMobileMenu)));
 /* Project Detail objects */
 const projectDetail = [{
-  title: 'Tonic',
-  exprience: ['CANOPY', 'Back End Dev', '2015'],
-  desription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
-  popDescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  title: 'Website',
+  exprience: ['Website', 'Front End Dev', '2020'],
```

Comment:
- [ ] Great job on adding a project to your website, since we have more than one project spot, I would suggest you add more. this will make your website more professional and attractive 😄 
- [ ] let make around to the design, I think there is some point of improvement in the desktop version, please consider adjusting it referring to this image below ⬇️ 
<img width="816" alt="tarik" src="https://user-images.githubusercontent.com/77843050/154860904-9c322aa8-2d80-4fbe-be68-4ac47abb602b.png">